ui/src/treeDataProvider/AzureAccountTreeItemBase.ts

```diff
@@ -32,6 +32,7 @@ export abstract class AzureAccountTreeItemBase extends AzExtParentTreeItem imple
     public readonly label: string = 'Azure';
     public childTypeLabel: string = localize('subscription', 'subscription');
     public autoSelectInTreeItemPicker: boolean = true;
+    public azureAccount: AzureAccount | undefined;
```

Comment:
This isn't actually exposed because you didn't update [index.d.ts](https://github.com/microsoft/vscode-azuretools/blob/master/ui/index.d.ts). That file is our public contract and affects whether or not something is "exposed".

More importantly, it's been a very conscious choice _not_ to expose this. The types for "AzureAccount" are defined [here](https://github.com/microsoft/vscode-azuretools/blob/master/ui/src/azure-account.api.d.ts#L13) and we don't want that in our public contract (index.d.ts) because we don't own those types. They are owned by the Azure Account extension [here](https://github.com/microsoft/vscode-azure-account/blob/master/src/azure-account.api.d.ts).